contrib/babelfishpg_tds/src/backend/tds/tds_data_map.c

```diff
@@ -95,7 +95,7 @@ TdsLCIDToEncodingMap TdsLCIDToEncodingMap_data[] =
 	{0x043e, PG_WIN1252}, //Malay:Malaysia
 	/* {0x044e, 0},// Marathi: India */
 	{0x0450, PG_WIN1251}, //Mongolian:Mongolia
-	{0x0414, PG_WIN1252}, //Norwegian:Norway(Bokm� � l)
+	{0x0414, PG_WIN1252}, //Norwegian:Norway(Bokm� � l)
```

Comment:
is this intended?

Reply:
No it wasn't intended might have been replaced by mistake, Resolved now